.isort.cfg

```diff
@@ -0,0 +1,9 @@
+[settings]
+line_length = 100
+indent = 4
+force_single_line = true
+case_sensitive = true
+# force_sort_within_sections = true
+default_section = THIRDPARTY
+sections = FUTURE,STDLIB,THIRDPARTY,FIRSTPARTY,LOCALFOLDER
+known_first_party = db,journalist_app,source_app,models,sdconfig,tests
```

Comment:
accidentally committed?

Reply:
[I've removed this file and rebased and squashed my commit, if we want to use isort we should open an issue and do in a separate PR, else you should add this file to your global gitignore]